python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -33,6 +33,7 @@
 
 heartbeat_times = None
 global_start_time = None
+global_count = 0
```

Comment:
Could we reset the `global_count` on shutdown so that it resets between tests etc.?